cmd/rcd/rcd.go

```diff
@@ -32,18 +41,209 @@ See the [rc documentation](/rc/) for more info on the rc flags.
 		if rcflags.Opt.Enabled {
 			log.Fatalf("Don't supply --rc flag when using rcd")
 		}
+
 		// Start the rc
 		rcflags.Opt.Enabled = true
 		if len(args) > 0 {
 			rcflags.Opt.Files = args[0]
 		}
+
+		if rcflags.Opt.WebUI {
+
+			// Get the latest release details
+			WebUIURL, tag, size := GetLatestReleaseURL()
+
+			// Load the file
+			exists := exists(tag + ".zip")
```

Comment:
This seems to be downloading and unzipping stuff in the current directory?

I'd suggest using the rclone cache directory instead `cachePath := filepath.Join(config.CacheDir, "webgui")`